improver/calibration/rainforest_calibration.py

```diff
@@ -639,31 +619,34 @@ def process(
             feature_cubes, forecast_cube
         )
 
-        # Evaluate the error CDF using tree-models.
-        error_CDF = self._calculate_error_probabilities(
+        # Evaluate the CDF using tree models.
+        per_member_CDF = self._calculate_threshold_probabilities(
             aligned_forecast, aligned_features
         )
 
+        # convert units of output thresholds
         if threshold_units:
-            original_threshold_unit = unit.Unit(threshold_units)
+            original_threshold_unit = Unit(threshold_units)
             forecast_unit = forecast_cube.units
-            output_thresholds_in_forecast_units = [
-                original_threshold_unit.convert(x, forecast_unit)
-                for x in output_thresholds
-            ]
+            output_thresholds_in_forecast_units = np.array(
+                [
+                    original_threshold_unit.convert(x, forecast_unit)
+                    for x in output_thresholds
+                ]
+            )
         else:
-            output_thresholds_in_forecast_units = output_thresholds
+            output_thresholds_in_forecast_units = np.array(output_thresholds)
 
         # Calculate probabilities at output thresholds
-        probabilities_by_realization = self._get_ensemble_distributions(
-            error_CDF, aligned_forecast, output_thresholds_in_forecast_units
+        interpolated_per_member_CDF = self._get_ensemble_distributions(
+            per_member_CDF, aligned_forecast, output_thresholds_in_forecast_units
         )
 
         # Average over realizations
-        output_cube = probabilities_by_realization.collapsed("realization", MEAN)
-        output_cube.remove_coord("realization")
+        probability_cube = interpolated_per_member_CDF.collapsed("realization", MEAN)
+        probability_cube.remove_coord("realization")
 
-        return output_cube
+        return probability_cube
```

Comment:
Could take this change one step further and change to `calibrated_probability_cube`.

Reply:
done